src/components/donate/Donate.js

```diff
@@ -1,7 +1,15 @@
 import React from "react";
+import Link from "next/link";
 
 const Donate = () => {
-  return <div>Donate</div>;
+  return (
+    <Link
+      href="#"
+      className="bg-green-600 text-white text-2xl font-bold text-decoration-line: underline flex justify-center items-center rounded-3xl min-w-fit px-4 min-h-fit py-1 hover:scale-110"
```

Comment:
no need for text-decoration-underline you can just do underline. Check tailwind docs. this code should work but the button does not look like then Figma. Change styling along with fonts, font size etc to make it look exactly like the Figma.